charts/oes/templates/spinnaker-extra/spinsvcs.yaml

```diff
@@ -15,12 +15,16 @@ metadata:
 spec:
   type: {{ .Values.k8sServiceType }}
   ports:
-  -  name: http
-     port: 9000
-     protocol: TCP
-  -  name: https
+   - name: "https"
      port: 443
      targetPort: 9000
+   - name: "http"
+     port: 80
+     protocol: TCP
+     {{- if .Values.global.spinDeck.port }}
+     nodePort: {{ .Values.global.spinDeck.port }}
```

Comment:
Let's also add a condition that if k8sServiceType is set to nodePort, then only add this